doctr/io/elements.py

```diff
@@ -253,6 +256,79 @@ def synthesize(self, **kwargs) -> np.ndarray:
 
         return synthesize_page(self.export(), **kwargs)
 
+    def export_as_xml(self, file_title: str = 'docTR - XML export (hOCR)', **kwargs) \
+            -> Tuple[bytes, ET.ElementTree]:
+        """Export the page as XML
+
+        Args:
+            return_plain: whether to return the plain (bytes) XML string or an ElementTree object
+            file_title: the title of the XML file
+            **kwargs: additional arguments to pass to the exporter
+
+        Returns:
+            the XML element
+        """
+        p_idx = self.page_idx
+        block_count: int = 1
+        line_count: int = 1
+        word_count: int = 1
+        width, height = self.dimensions
+        language = self.language if 'language' in self.language.keys() else 'en'
+        # Create the XML root element
+        page_hocr = ETElement('html', attrib={'xmlns': 'http://www.w3.org/1999/xhtml', 'xml:lang': str(language)})
+        # Create the header / SubElements of the root element
+        head = SubElement(page_hocr, 'head')
+        SubElement(head, 'title').text = file_title
+        SubElement(head, 'meta', attrib={'http-equiv': 'Content-Type', 'content': 'text/html; charset=utf-8'})
+        SubElement(head, 'meta', attrib={'name': 'ocr-system', 'content': f"python-doctr {doctr.__version__}"})
+        SubElement(head, 'meta', attrib={'name': 'ocr-capabilities',
+                                         'content': 'ocr_page ocr_carea ocr_par ocr_line ocrx_word'})
+        # Create the body
+        body = SubElement(page_hocr, 'body')
+        SubElement(body, 'div', attrib={
+            'class': 'ocr_page',
+            'id': f'page_{p_idx + 1}',
+            'title': f'image; bbox 0 0 {width} {height}; ppageno 0'
+        })
+        # iterate over the blocks / lines / words and create the XML elements in body line by line with the attributes
+        for block in self.blocks:
+            xmin, ymin, xmax, ymax = [coord for coordinates in block.geometry for coord in coordinates]
+            block_div = SubElement(body, 'div', attrib={
+                'class': 'ocr_carea',
+                'id': f'block_1_{block_count}',
```

Comment:
Why the "1" in "block_1_....." ?

Reply:
@fg-mindee 
I think this can be removed i can not see it in the convention (was mainly taken from the tesseract hOCR output).
Let me check this again with the parser